app/src/main/java/com/pokerio/app/utils/GameState.kt

```diff
@@ -560,6 +580,44 @@ object GameState {
     fun getMaxBet(): Int {
         return players.maxOf { it.bet }
     }
+
+    fun getSmallBlindPlayer(): Player? {
+        if (players.size < 2) {
+            return null
+        }
+        return players[players.size - 2]
+    }
+
+    fun getBigBlindPlayer(): Player? {
+        if (players.size < 2) {
+            return null
+        }
+        return players[players.size - 1]
+    }
+
+    fun nextCurrentPlayer() {
+        var index = players.indexOfFirst { player: Player -> player == currentPlayer }
+        if (index != -1) {
```

Comment:
Instead of the `if` statement, use `check(index != -1)`. Again I think this function should just fail and we want to handle the (essentially) bug somewhere else

Reply:
Done.